renderlib/StringUtil.cpp

```diff
@@ -74,3 +75,9 @@ splitToNameValuePairs(const std::string& s)
   }
   return pairs;
 }
+
+std::string
+escapePath(const std::string& path)
+{
+  return std::regex_replace(path, std::regex("\\\\"), "\\\\");
```

Comment:
Is this replacing `\\` with `\\\\`? Do you also need to worry at all about `\` characters?

Reply:
I realize it looks odd but the regex string itself has to be escaped, because of what backslash means in a regex.  so the regex is actually `\\` which means to find single backslash.  And I'm replacing it with `\\` a double backslash.  This function is used when I need to output a filepath as a python string to construct a python script..